roda-ui/roda-wui/src/main/java/org/roda/wui/client/ingest/process/PluginParameterPanel.java

```diff
@@ -7,14 +7,16 @@
  */
 package org.roda.wui.client.ingest.process;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Set;
+import java.util.*;
```

Comment:
Please do not use wildcard on imports. To do this on IntelliJ IDEA please use Eclipse Code Formatter plugin and add provided configurations: https://github.com/keeps/roda/tree/master/code-style